### ISSUE TITLE
[Achievement]Mercenary of Sholazar

### ISSUE THREAD

**Issue by alexuktc** [dalaranwow/dalaran-wow#4331]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
The easiest way is to explain what my experience has been. I have been exalted with The Oracles for over 10 months, getting an egg on cooldown. I got the Green Proto Drake so I decided to change to Frenzyheart and get the Mercenary of Sholazar Achievement. I did the Champion of Frenzyheart quest, which requires me to go kill Artius (the lich in the cave) and kill The Oracles npc. I correctly got the quest and it set me to Honored. I went to the Frenzyheart hub. They are friendly, but do not offer daily quests. Also any frenzyheart mobs outside of that hub are hostile. The oracles are hostile towards me but I cannot put "At War" with them on my reputation panel, meaning that even if I get my dailies working I will only be able to damage them with AoE (non targeted spells).

**Expected behaviour**: Tell us what should happen instead
Once I have completed the Champion of Frenzyheart Quest I have to be honored with them and I should have access to their daily quests. I should also be friendly to the mobs outside of the main faction hub and I should be able to attack mobs from The Oracles normally. 

**Steps to reproduce the problem**:

1. Get a reputation with one of the factions.
2.  Try to change the reputation
3.  See that you are not able to attack the other faction properly, you are not allied with all the mobs from that faction and that you are not given dailies. 

**Include proofs for this behaviour**
I am more than happy to talk to a GM in game and show him proof of what I am posting here. I actually made a ticket and was told I have not read properly, which I assure you is not correct. Contact Alekk in-game if you need further proof. 

http://www.wowhead.com/achievement=952/mercenary-of-sholazar#comments - Comments about how it's done on retail, including guides should it "bugg". I have done all that and nothing. Plus it does not apply as I am doing this on WotlK and cannot one shot the lich nor the other 2 npcs. 

Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
Jaloot (The Oracles nps) - 28667
Zepik the Gorloc Hunter (Frenzyheart npc) - 28668 

I don't believe I can include the IDs for the entire factions' npcs. 

https://www.youtube.com/watch?v=JrIbqsJCylY - Pretty much a video of what I did.

**Include Screenshots from the issue if necessary**
You can clearly see that I have the reputation needed with them and that since I am at 1/12000 I have not actually completed any dailies yet. I have the "Low Quest" tracking so there is no way I am not seeing them on the map. 
![bugg](https://cloud.githubusercontent.com/assets/24550199/22343503/f98480fc-e400-11e6-9cc9-fdfc3795e55e.png)

**Comment by Rushor** [comment]
add you details in this issue pls https://github.com/dalaranwow/dalaran-wow/issues/3507

/duplicate
